### ISSUE TITLE
Property 'media' does not exist on type '() => { media: Readable<{ [name: string]: boolean; }>; }'. Why?

### ISSUE THREAD

**Issue by frederikhors** [pearofducks/svelte-match-media#12]
First of all thanks for your work!

Using it like this for the first time today with TS:

```ts
import { createMedia } from "svelte-match-media";

const breakpoints = {
	sm: "screen and (max-width: 767px)",
	lg: "screen and (min-width: 1024px)",
};

const { media } = createMedia(breakpoints);

console.log("media:", media);
```
I get this error:

`Property 'media' does not exist on type '() => { media: Readable<{ [name: string]: boolean; }>; }'. ts(2339)`

**Why?** It's my fault?

![image](https://user-images.githubusercontent.com/41120635/111884442-f6f6e080-89c1-11eb-8508-e0e2c03b79b2.png)

**Comment by pearofducks** [comment]
Do you have a link to a reproduction repo? There's a lot of things that could be going wrong with TS to diagnose this.

The type definition _looks_ right FWIW.

**Comment by frederikhors** [comment]
> Do you have a link to a reproduction repo?

Here it is: https://github.com/frederikhors/svelte-match-media-issue-12

```
npm install
npm run validate
```

and you can read the error:

```
Error: Property 'media' does not exist on type '() => { media: Readable<{ [name: string]: boolean; }>; }'. (ts)

  const { media } = createMedia(breakpoints);
```

**Comment by pearofducks** [comment]
Fixed in 1.2.1 - thanks for reporting this!
